tools/translation_manager.py

```diff
@@ -1,38 +1,43 @@
 #!/usr/bin/env python3
 
+import json
+import copy
+import glob
+
 import fire
-import json, copy, glob
 
-LOCALE_DIR="../locales"
 
-INDENT=4
+LOCALE_DIR = "../locales"
+INDENT = 4
 
 
-def openLocaleFile(fname):
+def open_locale_file(fname):
     """Opens a locale file"""
-    with open(fname) as json_file: 
-      return json.load(json_file) 
+    with open(fname) as json_file:
+        return json.load(json_file)
+
 
-  
 def write_json(data):
-    return json.dumps(data, indent=INDENT, ensure_ascii=False, sort_keys=True) 
+    """Writes JSON to a string"""
+    return json.dumps(data, indent=INDENT, ensure_ascii=False, sort_keys=True)
 
 
 def export(data, fname):
     """Writes data to the correct locale file"""
-    with open(fname, 'w', encoding='utf-8') as json_file: 
-        json_file.write(write_json(data)) 
+    with open(fname, 'w', encoding='utf-8') as json_file:
+        json_file.write(write_json(data))
+
 
-        
-def is_expected(key, dest, expValue):
-    return key in dest and isinstance(dest[key], expValue)
+def is_expected(key, dest, exp_type):
+    """Returns true if key in dest and is instance of exp_type, e.g. configured as expected"""
```

Comment:
Thank you for this! Always good to have comments on return values.
I wonder if in the future we should adopt a specific standard formatting style, such as [Google doc comments](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods)? Completely a future concern